src/types.ts

```diff
@@ -32,8 +33,20 @@ export interface ZAuctionInstances {
   [registrarAddress: string]: zAuction.Instance;
 }
 
-export interface RouteUriToInstance {
-  [key: string]: ZAuctionInstances;
+export interface TokenAddressMapping {
+  [address: string]: Mapping;
```

Comment:
This should be `Maybe<Mapping>`

where `Maybe<T> = T | undefined`

`Maybe` should already be defined somewhere though